src/misc.jl

```diff
@@ -104,6 +133,18 @@ findat(a::AbstractArray, b::AbstractArray) = findat!(Array(Int, size(b)), a, b)
 # c: categories
 # k: the maximum integer in x
 
+"""
+    indicatormat(x::IntegerArray, k::Integer; sparse::Bool=false)
+
+### Args:
+* `x`: A, `IntegerArray`.
+* `k`: An `Integer`
+* `sparse`: A keyword argument, of `Bool` type.
+
+Construct a boolean matrix `r` of size `(k, length(x))` such that `r[x[i], i] = true` and all other elements are set to false.
+
+The keyword argument sparse controls whether to construct a sparse matrix. By default, it is `false`. 
```

Comment:
Missing ` around "sparse". Again, I don't think mentioning "keyword argument" is useful here.